deployment/CP04OSSM_Deploy.csv

```diff
@@ -86,86 +86,86 @@ AT-31,,AR-04,,CP04OSSM-RID27-03-CTDBPC000,3,1,2015-10-22T19:38:00,2016-05-13T17:
 AT-31,,AR-04,,CP04OSSM-RID27-04-DOSTAD000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGINS-DOSTAD-00503,39.93727,-70.88665,,7,454,
 AT-31,,AR-04,,CP04OSSM-SBC11-00-CPMENG000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGCON-BCPM01-00006,39.93727,-70.88665,,0,454,
 AT-31,,AR-04,,CP04OSSM-SBD11-00-DCLENG000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGCON-BDCL11-00006,39.93727,-70.88665,,0,454,
-AT-31,,AR-04,,CP04OSSM-SBD11-01-MOPAK0000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,OL000551,39.93727,-70.88665,,0,454,
-AT-31,,AR-04,,CP04OSSM-SBD11-02-HYDGN0000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,OL000552,39.93727,-70.88665,,0,454,
-AT-31,,AR-04,,CP04OSSM-SBD11-06-METBKA000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGINS-METLGR-00001,39.93727,-70.88665,,-3,454,
+AT-31,,AR-04,,CP04OSSM-SBD11-01-MOPAK0000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGINS-MOPAK0-99999,39.93727,-70.88665,,0,454,
+AT-31,,AR-04,,CP04OSSM-SBD11-02-HYDGN0000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGINS-HYDGN0-99999,39.93727,-70.88665,,0,454,
+AT-31,,AR-04,,CP04OSSM-SBD11-06-METBKA000,3,1,2015-10-22T19:38:00,2016-05-13T17:18:00,CGMCP-04OSSM-00003,,CGINS-METLGR-99991,39.93727,-70.88665,,0,454,
```

Comment:
Confirming changes to sensor UIDs and deployment depths for deployment 3. all look good.